src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -28,6 +28,7 @@
 #include <controller/CommissionerDiscoveryController.h>
 #include <platform/CHIPDeviceLayer.h>
 #include <setup_payload/AdditionalDataPayloadGenerator.h>
+#include <app/app-platform/ContentAppPlatform.h>
```

Comment:
This is failing to compile.

Reply:
I don't think this include is needed anymore

Reply:
Fixed.